bootstrap.sh

```diff
@@ -254,136 +233,25 @@ xcode_license() {
-  fi
-  (
-    cd "$HOMEBREW_PREFIX"
-    sudo_askpass mkdir -p Cellar Caskroom Frameworks bin etc include lib opt sbin share var
-    sudo_askpass chown "$USER:admin" Cellar Caskroom Frameworks bin etc include lib opt sbin share var
-  )
-
-  [ -d "$HOMEBREW_REPOSITORY" ] || sudo_askpass mkdir -p "$HOMEBREW_REPOSITORY"
-  sudo_askpass chown -R "$USER:admin" "$HOMEBREW_REPOSITORY"
-
-  if [ $HOMEBREW_PREFIX != $HOMEBREW_REPOSITORY ]; then
-    ln -sf "$HOMEBREW_REPOSITORY/bin/brew" "$HOMEBREW_PREFIX/bin/brew"
-  fi
-
-  # Download Homebrew.
-  export GIT_DIR="$HOMEBREW_REPOSITORY/.git" GIT_WORK_TREE="$HOMEBREW_REPOSITORY"
-  git init $Q
-  git config remote.origin.url "https://github.com/Homebrew/brew"
-  git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
-  git fetch $Q --tags --force
-  git reset $Q --hard origin/master
-  unset GIT_DIR GIT_WORK_TREE
-  logk
-
-  # Update Homebrew.
-  export PATH="$HOMEBREW_PREFIX/bin:$PATH"
-  logn "Updating Homebrew:"
-  [ -z "$QUICK" ] && brew update --quiet
+  sudo_askpass chown "$USER" /opt
+  where="/opt/homebrew"
+  [ -n "$CI" ] && where="/usr/local/Homebrew"
+  [ -d "$where" ] || git clone --depth=1 "https://github.com/Homebrew/brew" "$where"
+  export PATH="${where}/bin:${PATH}"
+  [ -z "$QUICK" ] && {
+    logn "Updating Homebrew"
+    brew update --quiet
+  }
   logk
 
-  # On Apple M1 machines we need to add this to the profile
-  if [[ "$(uname -m)" == "arm64" ]]; then
-    shell_profile=$(get_brew_profile)
-    eval "$(/opt/homebrew/bin/brew shellenv)"
-    if ! grep -qF "brew shellenv" "${shell_profile}"; then
-      #shellcheck disable=SC2016
-      echo -e 'eval "$(/opt/homebrew/bin/brew shellenv)"' >>${shell_profile}
-    fi
+  if ! grep -qF "brew shellenv" "${HOME}/.zshrc"; then
+    #shellcheck disable=SC2016
+    echo -e 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> "${HOME}/.zshrc"
   fi
-
-  # Install Homebrew Bundle, Cask and Services tap.
-  log "Installing Homebrew taps and extensions:"
-  brew bundle --quiet --file=- <<RUBY
-tap "homebrew/cask"
-tap "homebrew/core"
-tap "homebrew/services"
-RUBY
   logk
 }
 
-# Check and install any remaining software updates.
-software_update() {
-  logn "Checking for software updates:"
-  updates=$(softwareupdate -l 2>&1)
-  if echo "$updates" | grep "$Q" "No new software available."; then
-    logk
-  else
-    echo
-    if [ "$1" == "reminder" ]; then
-      log "You have system updates to install. Please check for updates if you wish to install them."
-      log "$updates"
-    elif [ -z "$CI" ]; then
-      log "Installing software updates:"
-      sudo_askpass softwareupdate --install --all
-      xcode_license
-    else
-      echo "Skipping software updates for CI"
-    fi
-    logk
-  fi
-}
-
-get_shell_name() {
-  case "$SHELL" in
-  /bin/bash)
-    echo "bash"
-    ;;
-  /bin/zsh)
-    echo "zsh"
-    ;;
-  esac
-}
-
-# From https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh
-get_brew_profile() {
```

Comment:
Why remove this? It seems like the right thing to do for folks who don't use zsh